File: SDKs/beezup.com/2.0/python/src/sdk/models/shared/storealertproperty.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class StoreAlertProperty:
    property_id: int = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('propertyId') }})
    property_value: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('propertyValue') }})
    

File: SDKs/beezup.com/2.0/typescript/src/sdk/models/shared/storealertproperty.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class StoreAlertProperty extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=propertyId" })
  propertyId: number;

  @SpeakeasyMetadata({ data: "json, name=propertyValue" })
  propertyValue: string;
}
